### PR TITLE
chore(cargo): update dependency

### PR DESCRIPTION
**Pull request by sdwoodbury**
#### Blink features:
- record call to mp4 file
- auto-mute: eliminates feedback from speakers to microphone. 
- volume control: make a peer sound louder/quieter
- use forks of opus related crates to allow proper cross compiling, needed for mobile builds

#### Other features:
- Multipass::import_identity

### Which issue(s) this PR fixes 🔨

- Resolve #

### Additional comments 🎤
`MultiPass::import_identity` does not have any functionality at this time, but will in the future

### CODE REVIEW

File: common/src/state/mod.rs [hunk 1]
```diff
@@ -604,6 +604,11 @@ impl State {
                 // todo
                 log::info!("audio degradation for peer {}", peer_id);
             }
+            BlinkEventKind::AudioOutputDeviceNoLongerAvailable
+            | BlinkEventKind::AudioInputDeviceNoLongerAvailable => {
+                // todo: notify user
+                log::info!("audio I/O device no longer available");
+            }
```
Comment (lgmarchi) [inline common/src/state/mod.rs:611]: Is it not better to let already 2 separate error treatment here?

**Review by lgmarchi — APPROVED** [review]

File: common/src/state/mod.rs (hunk 1 above)
Comment (sdwoodbury) [reply common/src/state/mod.rs:611]: yes - not better :grin: After all, the behavior is the same as far as the user is concerned and the task of handling those events is planned to be implemented in a separate PR.

**Review by dariusc93 — APPROVED** [review]
I did update warp to revert the initialization changes (already updated the commits here) but everything else should be good to go.

**Comment by phillsatellite** [comment]
Getting this error when trying to compile on windows, no issues when trying to compile on MacOS or Linux 

image

**Review by stavares843 — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**